# service type for saving image.
string path
int64 number
---
bool success
